src/github.com/stellar/federation/record.go

```diff
@@ -1,8 +1,12 @@
 package federation
 
-type Record struct {
-	Username  string `db:"username" json:"stellar_address"`
-	AccountId string `db:"account_id" json:"account_id"`
-	MemoType  string `db:"memo_type" json:"memo_type,omitempty"`
```

Comment:
the JSON response is the same but what you are fetching from the DB is different. 
`username` is weird. Is that the stellarAddress? Then we should call it that.